lib/validation.js

```diff
@@ -1,5 +1,13 @@
 const { INTERVAL_SHORTCUTS, ERL_QUOTA_INTERVALS_SHORTCUTS } = require('./utils');
 
+const ERL_BUCKET_KEYS = [
+  'size', 'per_interval', 'erl_activation_period_seconds', 'erl_quota_amount', 'erl_quota_interval'
+];
+
+const ERL_MANDATORY_KEYS = [
+  'size', 'per_interval', 'erl_activation_period_seconds', 'quota_per_calendar_month'
```

Comment:
I was confused by this too. Looking into it, it's because `ERL_MANDATORY_KEYS` is for the bucket config **before** parsing and `ERL_BUCKET_KEYS` is for the bucket config **after** parsing. That's also why `ERL_BUCKET_KEYS` does not fully match the docs, because it's our internal representation.

It's a bit confusing. I think we could simplify by validating the user input only. Wdyt?